src/components/views/directory/NetworkDropdown.js

```diff
@@ -170,8 +170,19 @@ export default class NetworkDropdown extends React.Component {
             icon = <img src="img/network-matrix.svg" width="16" height="16" />;
             span_class = 'mx_NetworkDropdown_menu_network';
         } else {
-            name = this.props.config.networkNames[network];
-            icon = <img src={this.props.config.networkIcons[network]} />;
+            if (this.props.config.networks[network]) {
```

Comment:
name is left `undefined` if this is false? would it make more sense to throw if it is false?